### PR TITLE
fix: handle organization level commands

### PR DESCRIPTION
**Pull request by a-cordier**
see https://github.com/gravitee-io/issues/issues/7408

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/fix-organization-level-commands/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_18_0/schema.yml [hunk 1]
```diff
@@ -9,3 +9,15 @@ databaseChangeLog:
...
                   name: environment_id
                   type: nvarchar(64)
+        - addColumn:
+            tableName: ${gravitee_prefix}commands
+            columns:
+              - column:
+                  name: organization_id
+                  type: nvarchar(64)
+                  constraints:
+                    nullable: false
```
Comment (gaetanmaisse) [inline gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_18_0/schema.yml:21]: Are we confident enough about that one?

File: gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/config/mock/CommandRepositoryMock.java [hunk 2]
```diff
@@ -95,5 +97,7 @@ void prepare(CommandRepository commandRepository) throws Exception {
         when(commandRepository.search(argThat(o -> o != null && o.isNotExpired()))).thenReturn(singletonList(search2));
         when(commandRepository.search(argThat(o -> o != null && "DEFAULT".equals(o.getEnvironmentId()))))
             .thenReturn(singletonList(newCommand));
+        when(commandRepository.search(argThat(o -> o != null && "DEFAULT".equals(o.getOrganizationId()))))
```
Comment (gaetanmaisse) [inline gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/config/mock/CommandRepositoryMock.java:100]: Ahhh these good old mocks, can't wait for the in memory version of the repository

**Review by gaetanmaisse — APPROVED** [review]

File: gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_18_0/schema.yml (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_18_0/schema.yml:21]: Ok, looks ok as all envs are related to an organization 🤞🏻

**Review by ThibaudAV — APPROVED** [review]

**Comment by a-cordier** [comment]
- [x] 🛑 as we are starting a quality sprint I will add the test on the CommandServiceImpl in order to make sonar happy

**Pull request merged**